sentrysql/operation_test.go

```diff
@@ -0,0 +1,50 @@
+package sentrysql
+
+import "testing"
+
+func TestParseDatabaseOperation(t *testing.T) {
+	tests := []struct {
+		name  string
```

Comment:
Just a suggestion, I prefer tests in format map[string]struct, where key is the name of the test. This makes it easier to read the name of the test (due to indentation), and prevents duplicate names.

Reply:
I actually don't like this way of testing (using `tests` array of struct and then iterate over it), I prefer something like `TestParseDatabaseOperation_CaseXXX` kind of thing.